tools/code-analyzer/src/lib/__tests__/fixtures/diff.txt

```diff
@@ -32494,7 +32494,7 @@ index 0000000000..ee57fbfa48
 +	 *
 +	 * ## EXAMPLES
 +	 *
-+	 *     # Install the latest version from woocommerce.com and activate
++	 *     # Install the latest version from woo.com and activate
```

Comment:
```suggestion
+	 *     # Install the latest version from Woo.com and activate
```

Reply:
For some reason GitHub doesn't let me apply this suggestion. I'll make a manual commit. 👍